Makefile

```diff
@@ -1,15 +1,15 @@
 serve:
-	hugo server \
+	npx hugo server \
 		--buildDrafts \
 		--buildFuture \
 		--disableFastRender
 
 production-build:
-	hugo \
+	npx hugo \
 		--minify
 
 preview-build:
-	hugo \
+	npx hugo \
```

Comment:
Why `npx` versus yarn or make?

Reply:
cc @chalin

Reply:
`npx` is what we use to access binaries fetched via an `npm` package.

Reply:
The broader question is whether we want to use the `hugo-extended` package at all. We should discuss that on Wednesday.